engine-rest/engine-rest/src/test/java/org/camunda/bpm/engine/rest/util/DateTimeUtils.java

```diff
@@ -40,6 +40,10 @@ public static String withTimezone(String dateString) {
     }
   }
 
+  public static String withTimezone(Date date) {
```

Comment:
I would also call this method in the `return` statement of the `withTimezone(String dateString)` method above (line 37), so we don't have duplicate methods.